### PR TITLE
added form validation files

### PR DESCRIPTION
**Pull request by DavyKn**
Added form validation files in my JS .

### CODE REVIEW

File: javascript/script.js [hunk 1]
```diff
@@ -62,4 +65,25 @@ for (let i = 0; i < myObjects.length; i += 1) {
...
+  event.preventDefault();
+  if (checkLowerCase(emailContactForm.value.trim())) {
+    contactForm.submit();
+    errorMessage.style.display = 'none';
+    emailContactForm.className = 'email';
+  } else {
+    errorMessage.style.display = 'block';
+    emailContactForm.className = '.error';
+  }
+});
```
Comment (aimemalaika) [inline javascript/script.js:89]: - [ ] Kindly check if a user submits the contact form with the email field containing only lowercase characters and the rest of the inputs are filled the form should be sent normally.

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @DavyKn , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights


## Required Changes ♻️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: javascript/script.js (hunk 1 above)
Comment (DavyKn) [reply javascript/script.js]: I have rectified the above-mentioned issue and it's now ok, Kindly let it be reviewed again

**Review by Donard97 — APPROVED** [review]
## Hi @DavyKn , 👋  

**Your project is complete!  There is nothing else to say other than... it's time to merge it 👏 💯**  

## Congratulations! 🎉 🎉 🎉 

giphy (7)

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**